src/screens/Customers/components/CustomerCard.tsx

```diff
@@ -0,0 +1,119 @@
+      flexGrow: 1,
+      display: 'flex',
+      alignItems: 'center',
+      paddingLeft: '17px',
+      color: theme.palette.text.primary,
+    },
+    totalText: {
+      color: theme.palette.text.secondary,
+    },
+    numberText: {
+      color: theme.palette.error.main,
+      marginLeft: '5px',
+    },
+    divSpacing: {
+      paddingLeft: '15px',
+    },
+    singleCard: {
+      width: '100%',
+      height: '93px',
+      border: `1px solid ${theme.palette.text.disabled}`,
+      borderRadius: '8px',
+      textAlign: 'left',
+      display: 'flex',
+      flexDirection: 'row',
+      marginBottom: '10px',
+    },
+    indicator: (props: CustomerCardProps) => ({
+      height: '7px',
+      width: '7px',
+      borderRadius: '50%',
+      backgroundColor: props.customer.isactive ? theme.palette.secondary.light : theme.palette.text.disabled,
+    }),
+    arrowSpacing: {
+      padding: '0px',
+    },
+    indicatorContainer: {
+      display: 'flex',
+      alignItems: 'center',
+      marginRight: '10px',
+    },
+    icon: {
+      borderRadius: '8px',
+      color: 'white',
+      padding: '5px',
+      backgroundColor: theme.palette.primary.main,
+      fontSize: '30px',
+    },
+  }),
+);
+
+interface CustomerCardProps {
+  customer: CustomerRecord;
+  match: any;
+  status?: CustomerStatus;
+}
+
+export default function CustomerCard(props: CustomerCardProps): JSX.Element {
+  const classes = useStyles(props);
+  const customer = props.customer;
+
+  const getStatusIcon = () => {
+    if (props.status === CustomerStatus.METER) {
+      return (
+        <IconButton
+          component={Link}
+          to={{ pathname: `${props.match.url}/customer/meter-readings/create` }}
+          size="small"
+          onClick={() => setCurrentCustomerIdInRedux(customer.id)}
+        >
+          <FlashOnIcon className={classes.icon} fontSize='inherit' />
+        </IconButton>
+      );
+    } else if (props.status === CustomerStatus.PAYMENT) {
+      return (
+        <IconButton
+          component={Link}
+          to={{ pathname: `${props.match.url}/customer/payments/create` }}
+          size="small"
+          onClick={() => setCurrentCustomerIdInRedux(customer.id)}
+        >
+          <AttachMoneyIcon className={classes.icon} fontSize='inherit' />
+        </IconButton>
+      );
+    }
+  }
+
+  return (
+      <Card className={classes.singleCard} variant="outlined">
+        <Link to={`${props.match.url}/customer`} onClick={() => setCurrentCustomerIdInRedux(customer.id)} className={classes.cardContent}>
+          <div className={classes.indicatorContainer}>
+            <div className={classes.indicator} />
+          </div>
+          <div>
+            <Typography variant="h2">{customer.customerNumber}, {customer.name}</Typography>
+            <Typography className={classes.totalText}>Total owed <span className={classes.numberText}>{customer.outstandingBalance} Ks</span></Typography>
+          </div>
+        </Link>
```

Comment:
You wanna wrap all of this in a `CardContent` tag with a `flexDirection: row` which will take care of the padding and flex height for you. You can see an example in the latest inventory cards in #87

Reply:
There is a lot of overlap with inventory cards, so I'd recommend checking out [this section](https://github.com/calblueprint/meepanyar/pull/87/files#diff-40f74751624d23ba073081b521ec85b80ad0a9a1735d72c67cedad290f10fb59R40-L48) where you can copy a lot of the structure and css.

Reply:
I tried to move over stuff but I don't really think CardContent adds much? It might make it even more dense because I have to make new divs & new styling to take care of the left/right sides with an indicator. Idk what you did though